packages/grafana-data/src/vector/FormattedVector.ts

```diff
@@ -1,10 +1,15 @@
 import { Vector } from '../types/vector';
 import { DisplayProcessor } from '../types';
 import { formattedValueToString } from '../valueFormats';
-import { vectorToArray } from './vectorToArray';
+import { FunctionalVector } from './FunctionalVector';
 
-export class FormattedVector<T = any> implements Vector<string> {
-  constructor(private source: Vector<T>, private formatter: DisplayProcessor) {}
+/**
+ * @public
+ */
+export class FormattedVector<T = any> extends FunctionalVector<string> {
```

Comment:
could not help myself ;)  extending FunctionalVector lets this get used in loops etc